nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/nxql/TestNxqlConversion.java

```diff
@@ -1696,6 +1700,48 @@ public void testConvertHintOperator() {
                 "  }\n" +
                 "}", es);
 
+        es = NxqlQueryConverter.toESQueryBuilder(
+                "select * from Document where /*+ES: OPERATOR(testTermQuery) */ ecm:uuid = '1234'").toString();
+        assertEqualsEvenUnderWindows("{\n" +
```

Comment:
Without adding `//` at the end of line you will have formatting issue.

Reply:
i agree, i wondering about doing it but the whole test don't use it. my question is can i modify just my new  tests or clean up the whole file